data/logs_model/table_logs_model.py

```diff
@@ -279,15 +282,19 @@ def load_logs():
                 )
 
                 logs_query = model.log.get_logs_query(
-                    namespace=namespace_id,
+                    namespace_id=namespace_id,
```

Comment:
Would it be more consistent to just use the actual namespace here instead of adding `namespace_id` as parameter to `_logs_query`. Just because every other methods in this class uses the namespace's name. Also if `_logs_query` is able to take the `namespace_id`, should it also be able to accept a `repository_id`, `performer_id`, ...

Reply:
We'd have to do another lookup, which is slower and less safe.